### PR TITLE
Add banner to documentation that announces beta release

### PR DESCRIPTION
**Pull request by brett0000FF**
**What does this PR do?**

Adds beta banner:
image

**Motivation:**

[DOCS-7564](https://datadoghq.atlassian.net/browse/DOCS-7564)

**Additional Notes:**

**How to test the change?**

**For Datadog employees:**
- [ ] If this PR touches code that signs or publishes builds or packages, or handles
credentials of any kind, I've requested a review from `@DataDog/security-design-and-guidance`.
- [x] This PR doesn't touch any of that.

Unsure? Have a question? Request a review!

[DOCS-7564]: https://datadoghq.atlassian.net/browse/DOCS-7564?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Datadog Ruby Trace Client
 
+<div class="alert alert-info">
+<strong>Beta</strong>: dd-trace-rb v2.0.0.beta1 is now available! This major release includes breaking changes, such as renaming the gem to <code>datadog</code> and extracting the CI visibility component to the <code>datadog-ci</code> gem. Please try it out and provide feedback.
```
Comment (alai97) [inline docs/GettingStarted.md:4]:
```suggestion
<strong>Beta</strong>: dd-trace-rb v2.0.0.beta1 is now available! This major release includes breaking changes, such as renaming the gem to <code>datadog</code> and extracting the CI Visibility component to the <code>datadog-ci</code> gem. Try it out and provide us with feedback.
```

**Review by alai97 — APPROVED** [review]
Looks good, one suggestion!

File: docs/GettingStarted.md (hunk 1 above)
Comment (ivoanjo) [inline docs/GettingStarted.md:4]: Should we perhaps link to https://github.com/DataDog/dd-trace-rb/blob/2.0/docs/UpgradeGuide2.md ?

Comment (brett0000FF) [reply docs/GettingStarted.md]: Good idea--added that in the latest commit. 👍

**Comment by TonyCTHsu** [comment]
Hey, we actually forgot to update Upgrade guide 2 to include the renaming. Can we postpone it with our beta2 release?

**Review by tushar-datadog — APPROVED** [review]

**Comment by brett0000FF** [comment]
@ivoanjo We are ready to merge this! We removed the sentence about the upgrade guide for now. We can add it back once the upgrade guide includes details about the renamed components. Thanks!

**Comment by marcotc** [comment]
I've just pushed this to the [`release`](https://github.com/DataDog/dd-trace-rb/tree/release) branch in order for the official documentation to pick it up.

**Pull request merged**